truffle.js

```diff
@@ -55,7 +55,11 @@ module.exports = {
     solc: {
       version: packageJson.devDependencies.solc,
       settings: {
-        evmVersion: 'istanbul'
+        evmVersion: 'istanbul',
+        optimizer: {
+          enabled: true,
+          runs: 200 // Optimize for how many times you intend to run the code
```

Comment:
Does it take noticeably more time to compile with optimizer?

Reply:
No, from what I saw